org.mixedrealitytoolkit.input/Interactors/HandJointInteractor.cs

```diff
@@ -49,7 +68,21 @@ public abstract class HandJointInteractor :
         public override bool isHoverActive
         {
             // Only be available for hovering if the controller is tracked or we have joint data.
-            get => base.isHoverActive && (xrController.currentControllerState.inputTrackingState.HasPositionAndRotation() || interactionPointTracked);
+            get
+            {
+#pragma warning disable CS0618 // XRBaseController is obsolete
+                if (forceDeprecatedInput)
+                {
+                    return base.isHoverActive && (xrController.currentControllerState.inputTrackingState.HasPositionAndRotation() || interactionPointTracked);
+                }
+#pragma warning restore CS0618 // XRBaseController is obsolete
+                else
+                {
+                    TrackedPoseDriver trackedPoseDriver = transform.GetComponentInParent<TrackedPoseDriver>();
+                    bool hasPositionAndRotation = ((InputTrackingState)trackedPoseDriver.trackingStateInput.action.ReadValue<int>()).HasPositionAndRotation();
```

Comment:
Should we include protection against null here, for both trackedPoseDriver (some custom prefab not using TrackedPoseDriver in parent) and tracking state input action (action not defined on a parent TrackedPoseDriver)?

Reply:
will do, thx for the feedback

Reply:
@whebertML I'll be closing this PR because a [new one](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824) was created since several changes were made while retaking this and it was easier to re-do the changes in the [new PR](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824).  Your feedback was attended in the new PR btw, thank you for providing the feedback.